pex/bin/pex.py

```diff
@@ -523,8 +551,9 @@ def main():
 
     log('Running PEX file at %s with args %s' % (pex_builder.path(), cmdline), v=options.verbosity)
     pex = PEX(pex_builder.path(), interpreter=pex_builder.interpreter)
-    sys.exit(pex.run(args=list(cmdline)))
+    raise SystemExit(pex.run(args=list(cmdline)))
```

Comment:
No strong objection.  I'll update the code to use sys.exit rather than raise SystemExit.